src/components/PlotSelector/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import { Grid, Typography, useTheme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { Autocomplete } from '@terraware/web-components';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import strings from 'src/strings';
+
+const useStyles = makeStyles(() => ({
+  horizontal: {
+    '& .MuiAutocomplete-root': {
+      width: '100%',
+    },
+  },
+}));
+
+export type PlotInfo = {
+  id: number;
+  fullName: string;
+};
+
+export type ZoneInfo = {
+  id: number;
+  name: string;
+  plots?: PlotInfo[];
+};
+
+export type PlotSelectorProps = {
+  zones: ZoneInfo[];
+  onPlotSelected: (plot?: PlotInfo) => void;
+  onZoneSelected?: (zone?: ZoneInfo) => void;
+  zoneError?: string;
+  plotError?: string;
+  horizontalLayout?: boolean;
```

Comment:
Where is this coming from?

Reply:
This will be used in the plots chart . Added this in the PR description.
Have asked Rachel if we can keep existing autocomplete layout let’s see. For now we need this for next task.